commons/all/src/main/java/org/infinispan/commons/marshall/Ids.java

```diff
@@ -50,7 +50,7 @@
    int XID_DEADLOCK_DETECTING_GLOBAL_TRANSACTION = 33;
    int XID_GLOBAL_TRANSACTION = 34;
 
-   int IN_DOUBT_TX_INFO = 35;
+   //int IN_DOUBT_TX_INFO = 35;
```

Comment:
why comment instead of remove ?

Reply:
I'll invoke @ryanemerson .
As far as I can see, we never remove the ids. we just left them there forever. 
Can they be removed Ryan?